Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -441,6 +441,19 @@ public struct AccessibilityIdentifiers {
                 static let close = "Close"
                 static let save = "Save"
             }
+
+            struct ViewCard {
```

Comment:
Nit: could this be more specific, similar to the next Struc, ViewCreditCard ?

Reply:
Updated

Reply:
> The struct is named based on the name of the screen. I followed this rule for every screen.

Oh I did not know that. Please use the name that suits better here. Thanks